src/formulate.app/Forms/Handlers/Email/EmailHandler.cs

```diff
@@ -452,7 +452,10 @@ public virtual void HandleForm(FormSubmissionContext context, object configurati
             if (isHtml)
             {
                 baseMessage = WebUtility.HtmlEncode(baseMessage);
-                lines = lines.Select(x => WebUtility.HtmlEncode(x)).ToList();
+
+                //Add line breaks in after encoding.
+                lines = lines.Select(x => WebUtility.HtmlEncode(x)
+                .Replace(Environment.NewLine, Environment.NewLine + "<br>" + Environment.NewLine)).ToList();
```

Comment:
It would be better to use a regex to do the replacement since line endings are different with different operating systems (and since `Environment.NewLine` will just use the server operating system's line break rather than the browser operating system's line break). Something like `(\r\n|\r(?!\n)|(?<!\r)\n)` would probably work. You'd want to make it compiled to ensure good performance.

Reply:
Hmm... I just used the same logic here that was used on line 371?

```
   var nl = isHtml
   ? Environment.NewLine + "<br>" + Environment.NewLine
   : Environment.NewLine;
```

Reply:
I've undone the version increment though (perhaps a little overeager there!)

Reply:
The line 371 you're referring to is not the same logic, as it isn't matching existing text to replace something. It's the matching portion that should be addressed, not the replacement (feel free to replace with `Environment.NewLine` as you're already doing).